README.md

```diff
@@ -33,8 +33,11 @@ Download binaries of **[Harbor release ](https://github.com/vmware/harbor/releas
 Refer to **[User Guide](docs/user_guide.md)** for more details on how to use Harbor.
 
 ### Community
+**Twitter:** @project_harbor  
+**User Group:** Join Harbor user email group: harbor-users@googlegroups.com to get update of Harbor's news, features, releases, or to provide suggestion and feedback. To subscribe, send an email to harbor-users+subscribe@googlegroups.com .  
```

Comment:
also link to mailto:harbor-users+subscribe@googlegroups.com :)

Reply:
It will change link automatically , you can view the result here https://github.com/hainingzhang/harbor/blob/05d8b1194e84a0a38b56a35f5ddebf6c1e431432/README.md